src/player.cpp

```diff
@@ -4099,10 +4102,10 @@ void PowerUps(const int A)
                             NPC[numNPCs].Type = 292;
                             NPC[numNPCs].Special5 = A;
                             NPC[numNPCs].Special6 = p.Direction;
-                            PlaySound(SFX_Throw);
+                            PlaySound(playerHammerSFX);
                         }
                         else
-                            PlaySound(playerHammerSFX);
+                            PlaySound(SFX_Throw);
```

Comment:
and `PlaySound(playerHammerSFX);` here. The `playerHammerSFX` is a global variable which is set by `sounds.ini`.